app/views/layouts/application.html.haml

```diff
@@ -1,7 +1,5 @@
-- if @mks || @vmrc || @vnc || controller.action_name == "menutest"
-  = yield
```

Comment:
> Do we want to wrap console sessions in our full header/footer layout?
no
> This will result in duplicate sets of html, header, body, footer tags in the resulting markup
no, those are rendered w/o a layout